### PR TITLE
Update links to images in FORGE-QUICKSTART.md

### PR DESCRIPTION
**Pull request by cwperks**
### Background

Currently, the images in [FORGE-QUICKSTART.md](https://github.com/Significant-Gravitas/AutoGPT/blob/master/classic/FORGE-QUICKSTART.md) are broken. This PR updates the links so that the images render properly.

Same markdown on my fork where the paths resolve correctly: https://github.com/cwperks/AutoGPT/blob/upadte-img-links/classic/FORGE-QUICKSTART.md

This is my first PR so I'm not aware of the history of the repo, but the primary issue is that FORGE-QUICKSTART.md is in the `classic/` folder and uses relative paths to the images which are under the root level `docs/` folder.

### CODE REVIEW

File: autogpt_platform/README.md [hunk 1]
```diff
@@ -28,9 +28,9 @@ To run the AutoGPT Platform, follow these steps:
 
 3. Run the following command:
    ```
-   cp supabase/docker/.env.example .env
+   cp supabase/docker/.env.example .env.local
    ```
-   This command will copy the `.env.example` file to `.env` in the `supabase/docker` directory. You can modify the `.env` file to add your own environment variables.
+   This command will copy the `.env.example` file to `.env.local` in the `supabase/docker` directory. You can modify the `.env.local` file to add your own environment variables.
```
Comment (majdyz) [inline autogpt_platform/README.md]: I think this is unrelated to the documentation described on the PR, can we revert this?

**Review by majdyz — CHANGES_REQUESTED** [review]

File: autogpt_platform/README.md (hunk 1 above)
Comment (cwperks) [reply autogpt_platform/README.md]: I initially targeted this to the master branch and GH changed the target branch to dev. I just rebased the branch to dev.

**Review by ntindle — APPROVED** [review]
